mixer/mixer_test.go

```diff
@@ -0,0 +1,183 @@
+package mixer
+
+import (
+	"os"
+	"testing"
+
+	"github.com/NIHERASE/voxcox/compressor"
+
+	"github.com/go-audio/audio"
+	"github.com/go-audio/wav"
+)
+
+func TestOneChanelInput(t *testing.T) {
+	ch1, ch2, exit2, output := chanelInitializer()
+
+	m := new(Mixer)
+	compressor := new(compressor.MockCompressorImpl)
+	m.audioCompressor = compressor
```

Comment:
`Mixer` does not allow this private field to be set by other packages. Tests shouldn't do it either.
This is another reason why it's better to put tests in a separate `_test` package.